packages/Button/stories/Button.mdx

```diff
@@ -0,0 +1,234 @@
+
+<Button isActive>active</Button>
+<Button isDisabled>disabled</Button>
+<Button isPending>pending</Button>
+
+#### Submit
+
+To turn a Button into a submit Button by adding the `isSubmit` property.
+
+```js
+<form
+  onSubmit={event => {
+    alert("submitted");
+    event.preventDefault();
+  }}
+>
+  <input type="text" defaultValue="press enter while focus" />
+  <Button isSubmit isSemantic={false}>
+    submit
+  </Button>
+</form>
+```
+
+##### Result
+
+<ButtonSubmit />
+
+### Close Button
+
+To make use of a Close Button use the `<Button.Close>` element.
+
+```js
+<p>
+<Button.Close onClick={clickHandler} ref={buttonRef} />
+</p>
+<p>
+<Small>This CloseButton will capture the focus after 1 second.</Small>
+</p>
+```
+
+##### Result
+
+<CloseButtonRef />
+
+### Link Button
+
+To make use of a Link Button use the `<Link.Button>` element.
+
+```js
+<Button.Link kind="primary" href="https://youtu.be/IdkCEioCp24?t=92">
+    Link
+</Button.Link>
+<Button.Link kind="secondary" href="https://youtu.be/IdkCEioCp24?t=92" shouldOpenNewTab>
+    Link in new tab
+</Button.Link>
+<Button.Link href="https://youtu.be/IdkCEioCp24?t=92" size="small">
+    Link small
+</Button.Link>
+<Button.Link href="https://youtu.be/IdkCEioCp24?t=92">Link medium</Button.Link>
+<Button.Link href="https://youtu.be/IdkCEioCp24?t=92" size="large">
+    Link large
+</Button.Link>
+<Button.Link href="https://youtu.be/IdkCEioCp24?t=92" shouldOpenNewTab>
+    Link in new tab
+</Button.Link>
+<Button.Link kind="secondary" icon={<PlusIcon />} href="https://youtu.be/IdkCEioCp24?t=92" shouldOpenNewTab>
+    Button link with icon
+</Button.Link>
+```
+
+##### Result
+
+<Button.Link kind="primary" href="https://youtu.be/IdkCEioCp24?t=92">
+    Link
+</Button.Link>
+<Button.Link kind="secondary" href="https://youtu.be/IdkCEioCp24?t=92" shouldOpenNewTab>
+    Link in new tab
+</Button.Link>
+<Button.Link href="https://youtu.be/IdkCEioCp24?t=92" size="small">
+    Link small
+</Button.Link>
+<Button.Link href="https://youtu.be/IdkCEioCp24?t=92">Link medium</Button.Link>
+<Button.Link href="https://youtu.be/IdkCEioCp24?t=92" size="large">
+    Link large
+</Button.Link>
+<Button.Link href="https://youtu.be/IdkCEioCp24?t=92" shouldOpenNewTab>
+    Link in new tab
+</Button.Link>
+<Button.Link kind="secondary" icon={<PlusIcon />} href="https://youtu.be/IdkCEioCp24?t=92" shouldOpenNewTab>
+    Button link with icon
+</Button.Link>
+
+### Icon Button
+
+The Icon Button allows users to easily make use of buttons that contain icons or use icons as buttons.
+
+```js
+<Button.Icon  size="small">
+    <InfoIcon />
+</Button.Icon>
+<Button.Icon >
+    <InfoIcon />
+</Button.Icon>
+<Button.Icon  size="large">
+    <InfoIcon />
+</Button.Icon>
+<Button.Icon  kind="flat">
+    <InfoIcon />
+</Button.Icon>
+<Button.Icon  kind="minor">
+    <InfoIcon />
+</Button.Icon>
+<Button.Icon  icon="🐐" kind="secondary" size="small">
+    <span role="img" aria-label="Llama">
+    🦙
+    </span>
+</Button.Icon>
```

Comment:
Let's cut this one ✂️ 
It's sort of a silly edge case test, not how we _should_ use the component.